resources/mod_proxy.rb

```diff
@@ -3,7 +3,7 @@
          description: ''
 
 property :require, String,
-         default: '',
+         default: 'all denied',
          description: ''
```

Comment:
We could do with a description here. And then copy that description to the documentation (which doesn't exist yet 😭 )